xml/article_nfs_storage.xml

```diff
@@ -379,68 +385,66 @@ include /etc/drbd.d/*.res;</screen>
 &prompt.crm.conf;<command>rsc_defaults resource-stickiness="200"</command>
```

Comment:
Is this section (Adjusting Pacemaker's configuration) still required?

Reply:
Good catch. I have the same comment and do suggest to delete it. 

Loop in @gao-yan to double check his view.